setup.py

```diff
@@ -277,6 +296,24 @@ def wrapper(*args, **kw):  # type: ignore
     _get_start_shard, lru_size=SLOTS_PER_EPOCH * 3)'''
 
 
+LIGHTCLIENT_PATCH_SUNDRY_FUNCTIONS = '''
```

Comment:
"Sundry" as in "miscellaneous" or as in "tomatoes"? :smile: 

(I admit I have seen that word used very little and I had to google to find the non-tomato-related meaning, so perhaps other people also have not seen the word much, and it may be worth replacing with another one)

Reply:
😂
I'm no tomatoes expert.
I guess @protolambda first named it? Does renaming it to `*_MISC_FUNCTIONS` make sense to you?

Reply:
Not a tomato expert either, I didn't name it. It's fine as-is though, unique names are useful, and "misc" isn't more descriptive.